remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/builder/datastore/BifrostLogUploaderProducer.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2022 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.remotecoordinator.builder.datastore;
+
+import org.jboss.pnc.auth.KeycloakServiceClient;
+import org.jboss.pnc.bifrost.upload.BifrostLogUploader;
+import org.jboss.pnc.common.json.GlobalModuleGroup;
+import org.jboss.pnc.common.json.moduleconfig.SystemConfig;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
+import java.net.URI;
+
+@ApplicationScoped
+public class BifrostLogUploaderProducer {
+
+    BifrostLogUploader logUploader;
+
+    @Inject
+    public BifrostLogUploaderProducer(
+            GlobalModuleGroup globalConfig,
+            SystemConfig systemConfig,
+            KeycloakServiceClient keycloakServiceClient) {
+        logUploader = new BifrostLogUploader(
+                URI.create(globalConfig.getExternalBifrostUrl()),
+                systemConfig.getBifrostLogUploadMaxRetries(),
+                systemConfig.getBifrostLogUploadRetryDelay(),
+                keycloakServiceClient::getAuthToken);
+    }
+
+    // @Produces TODO: enable when build-coordinator module is deleted
```

Comment:
Can you make it the other way? So that build-coordinator's version is commented and remote-coordinator is not. That way, removing build-coordinator won't require remembering that this TODO exists.